src/DynamoCoreWpf/Views/Core/DynamoView.xaml.cs

```diff
@@ -768,11 +768,13 @@ private void DynamoViewModelChangeScaleFactor(object sender, EventArgs e)
             var view = new Prompts.ChangeScaleFactorPrompt(dynamoViewModel.ScaleFactorLog) { Owner = this };
             if (view.ShowDialog() == true)
             {
-                if (dynamoViewModel.ScaleFactorLog != view.SliderValue)
+                if (dynamoViewModel.ScaleFactorLog != view.ScaleValue)
                 {
-                    dynamoViewModel.ScaleFactorLog = view.SliderValue;
+                    dynamoViewModel.ScaleFactorLog = view.ScaleValue;
                     dynamoViewModel.CurrentSpace.HasUnsavedChanges = true;
 
+                    Log("Scale factor log changed to " + view.ScaleValue);
```

Comment:
These numbers should probably be formatted using the user's locale, and not hardcoded.  For example, in "en-us": 
"0.000,001" There should not be a comma after the decimal point.
"1000,000" There should be a comma after the 1 as well.